main.go

```
@@ -41,79 +47,212 @@ func main() {
+}
+
+func subscribeCMD(ctx context.Context, pub *soccer.ChampionsLeague, bot *tApi.BotAPI, update *tApi.Update, _ *logrus.Logger) {
+	resp := tApi.NewMessage(update.Message.Chat.ID, "")
+	sub := pub.AddConsumer(int(resp.ChatID))
+
+	for {
+		select {
+		case ls, ok := <-sub.Event():
+			if !ok {
+				resp.Text = "channel closed"
+				bot.Send(resp)
```

golangci-lint (1.64.8)

Error return value of `bot.Send` is not checked

(errcheck)